### PR TITLE
[PROPOSAL-3139] Datasource selection changes from radio to select

### PR DESCRIPTION
**Pull request by wuchunfu**
## What is the purpose of the pull request

Datasource selection changes from radio to select. #3139 

## Brief change log

  - Add selectInput.vue file.
  - Modified the createDataSource.vue file

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

File: dolphinscheduler-ui/src/js/conf/home/pages/datasource/pages/list/_source/createDataSource.vue [hunk 1]
```diff
@@ -465,7 +494,7 @@
 
     mounted () {
     },
-    components: { mPopup, mListBoxF }
+    components: { mPopup, mListBoxF, mSelectInput }
```
Comment (yangyichao-mango) [inline dolphinscheduler-ui/src/js/conf/home/pages/datasource/pages/list/_source/createDataSource.vue:497]: Hi,
What is the purpose of introducing this component?
If you want to add drop-down filtering options, Ans-UI native is supported.
image

Comment (wuchunfu) [reply dolphinscheduler-ui/src/js/conf/home/pages/datasource/pages/list/_source/createDataSource.vue]: Sorry, this is my question. Thank you very much for your code review.

**Comment by yangyichao-mango** [comment]
+1

**Comment by qiaozhanwei** [comment]
@break60

**Comment by break60** [comment]
I think the difference between changing and changing is not very big. Selecting the radio button allows users to easily understand more data, but there are too many values here, beyond the scope of the page display, you can also change to select

**Pull request merged**